server/src/metadata.rs

```diff
@@ -69,6 +69,7 @@ pub mod entities_metadata_query {
         is_part_of: Vec<LinkedNode>,
         learning_resource_type: Vec<LinkedNode>,
         license: LinkedNode,
+        main_entity_of_page: serde_json::Value,
```

Comment:
```suggestion
        main_entity_of_page: Value,
```

Isn't `Value` already in scope? By the way `serde_json::Value` might be better since it is more descriptive...

Reply:
Value is not in scope. We could change that but yes serde_json::Value might be clearer anyway.